styles.css

```diff
@@ -146,17 +146,26 @@
     margin-top: 20px;
 }
 
-.qpy-show-more div {
+.qpy-card-description {
     text-align: justify;
 }
 .qpy-show-more .collapse:not(.show) {
     display: block;
     height: 4rem;
     overflow: hidden;
+    -webkit-mask-image: linear-gradient(to bottom, black 75%, transparent 100%);
+    mask-image: linear-gradient(to bottom, black 75%, transparent 100%);
 }
 
 .qpy-show-more .collapsing {
     height: 4rem;
+    -webkit-mask-image: linear-gradient(to bottom, black 75%, transparent 100%);
+    mask-image: linear-gradient(to bottom, black 75%, transparent 100%);
```

Comment:
Es wäre schön, wenn diese drei Zeilen nicht dupliziert wären. Du könntest die gleichzeitig auf den `.collapse:not(.show)` und den `.collapsing` Selektor anwenden und dann `display: block` und `overflow: hidden` nochmal separat.